velocity-transition-group.js

```diff
@@ -281,27 +283,39 @@ var VelocityTransitionGroup = React.createClass({
     // animations) so that we can then animate out. Velocity typically makes these transitions
     // very smooth, correctly animating from whatever state the element is currently in.
     if (entering) {
-      completeFn();
-      completeFn = null;
+      doneFn();
+      doneFn = null;
     } else {
       Velocity(nodes, 'stop');
     }
 
+    var combinedCompleteFn;
+    if (doneFn && opts.complete) {
+      var optsCompleteFn = opts.complete;
+      combinedCompleteFn = function () {
+        doneFn();
+        optsCompleteFn();
+      };
+    } else {
+      // One or the other or neither.
+      combinedCompleteFn = doneFn || opts.complete;
```

Comment:
I don't quite understand how passing `{complete: undefined}` will change the landscape of things, since we still say `doneFn || opts.complete` here, so doneFn would still be executed. Can/does the parsedAnimation object below sometimes pass in a non-null `complete`, hence us overriding it with `undefined`?